javascript/javascript1/week4/Voice_assistant.js

```diff
@@ -0,0 +1,114 @@
+
+let name = null;
+let toDoList = {};
+let timer = 15;
+
+function getReply(command){
```

Comment:
High level recommendation: 
Implementation of the task is correct, though there are several things that might make it better:
one thing among them is structure:
At this moment it's already quite complicated: a lot of ifs, elses. They are nested one into another. 
There are lot of logic here.
It worth to cut it into smaller functions like it's done with `startTimer`